code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -71,8 +71,8 @@
 		/obj/item/reagent_containers/food/snacks/grown/watermelon = list("watermelonjuice" = 0),
 		/obj/item/reagent_containers/food/snacks/watermelonslice = list("watermelonjuice" = 0),
 		/obj/item/reagent_containers/food/snacks/grown/berries/poison = list("poisonberryjuice" = 0),
-		/obj/item/reagent_containers/food/snacks/grown/pumpkin = list("pumpkinjuice" = 0),
 		/obj/item/reagent_containers/food/snacks/grown/pumpkin/blumpkin = list("blumpkinjuice" = 0),
+		/obj/item/reagent_containers/food/snacks/grown/pumpkin = list("pumpkinjuice" = 0),
```

Comment:
Can you add a comment telling the reader why the order is important here? I also noticed line 77 and 78. The lather probably isn't needed right?